README.md

```diff
@@ -151,10 +151,10 @@
       - [Add Builds to Indexing Configuration](#add-builds-to-indexing-configuration)
       - [Request Graph Scan](#request-graph-scan)
       - [Retrieve the Graph Scan Results](#retrieve-the-graph-scan-results)
-      - [Generate Vulnerabilities Report](#generate-vulnerabilities-report)
-      - [Get Vulnerabilities Report Details](#get-vulnerabilities-report-details)
-      - [Get Vulnerabilities Report Content](#get-vulnerabilities-report-content)
-      - [Delete Vulnerabilities Report](#delete-vulnerabilities-report)
+      - [Generate Vulnerabilities/Licences/Violations Report](#generate-vulnerabilities-licences-violations-report)
+      - [Get Vulnerabilities/Licenses/Violations Report Details](#get-vulnerabilities-licences-violations-report-details)
+      - [Get Vulnerabilities/Licenses/Violations Report Content](#get-vulnerabilities-licences-violations-report-content)
+      - [Delete Vulnerabilities/Licenses/Violations Report](#delete-vulnerabilities-licences-violations-report)
```

Comment:
@ns-dsinghal the pre-existing links (above/below your change) goto sections that describe more about the api. I think similar sections needs to be added for the new lines you added.
Also I think the name of the links must be for a specific api (i.e. not include words separated by `/`)

Reply:
Fixed